src/components/DataTable/Table.js

```diff
@@ -48,6 +50,11 @@ Table.propTypes = {
    */
   short: PropTypes.bool,
 
+  /**
+   * `false` Applies styles for data tables with sorting functionality.
+   */
+  sortable: PropTypes.bool,
```

Comment:
```suggestion
  isSortable: PropTypes.bool,
```

To match [`TableHeader`](https://github.com/IBM/carbon-components-react/blob/master/src/components/DataTable/TableHeader.js#L123)

Reply:
👍